### PR TITLE
Fix/0411

### PR DESCRIPTION
**Pull request by hugh-n**

### CODE REVIEW

File: app/views/admin/companies/_form.html.haml [hunk 1]
```diff
@@ -120,7 +120,7 @@
     .p-i-add-company.form-group
       %label.col-sm-2.control-label{for: "company_email"}請求先用メール
       .col-sm-10
-        = text_field :company, :billing_email, class: "form-control"
+        = text_field :company, :billing_email, class: "form-control", type: "email"
```
Comment (ericgpks) [inline app/views/admin/companies/_form.html.haml]: I think you can use this "email_field".
http://railsdoc.com/references/email_field_tag

File: app/views/admin/companies/_form.html.haml [hunk 2]
```diff
@@ -158,7 +158,7 @@
             .p-i-add-company.form-group
               %label.col-sm-2.control-label{for: "company_member_email"}メールアドレス
               .col-sm-10
-                = member.text_field :email, class: "form-control"
+                = member.text_field :email, class: "form-control", type: "email"
```
Comment (ericgpks) [inline app/views/admin/companies/_form.html.haml]: I think you can use this "email_field".
http://railsdoc.com/references/email_field_tag

File: app/views/admin/companies/_member_form.html.haml [hunk 3]
```diff
@@ -23,4 +23,4 @@
   .p-i-add-company.form-group
     %label.col-sm-2.control-label{for: "company_member_email"}メールアドレス
     .col-sm-10
-      = text_field_tag "company[company_members_attributes][#{@count}][email]", nil, class: "form-control"
\ No newline at end of file
+      = text_field_tag "company[company_members_attributes][#{@count}][email]", nil, class: "form-control", type: "email"
```
Comment (ericgpks) [inline app/views/admin/companies/_member_form.html.haml]: I think you can use this "email_field".
http://railsdoc.com/references/email_field_tag

File: app/controllers/users/download_controller.rb [hunk 4]
```diff
@@ -53,13 +53,14 @@ def download
...
+    Zip.unicode_names = true 
     Zip::OutputStream.open(t.path) do |zos|
       @businesses.each do |attachment|
         if attachment.document_url.present?
         # Create a new entry with some arbitrary name
           zos.put_next_entry(attachment.document.file.filename)
-          @path = URI.decode(attachment.document_url).force_encoding('UTF-8')
-          buffer = File.open("#{Rails.root}/public#{@path}", "rb", encoding:'utf8') { |f| f.read }
+          @path = URI.decode(attachment.document_url).force_encoding('Cp932')
+          buffer = File.open("#{Rails.root}/public#{@path}", "rb", encoding:'utf-8') { |f| f.read }
```
Comment (ericgpks) [inline app/controllers/users/download_controller.rb]: 💚

File: app/assets/stylesheets/object/user/service/form.scss [hunk 5]
```diff
@@ -161,6 +167,29 @@
...
+    box-shadow: 0px 0px 0px 0px rgba(0, 0, 0, 0.4);
+    color: #fff;
+    font-size: 18px;
+    font-weight: 700;
+    height: 52px;
+    margin-bottom: 30px;
+    margin-left: 40px;
+    margin-top: 20px;
+    padding: 10px 65px;
+    width: 260px;
```
Comment (ericgpks) [inline app/assets/stylesheets/object/user/service/form.scss]: Please fix the order.

File: app/assets/stylesheets/object/user/article/article.scss [hunk 6]
```diff
@@ -124,9 +124,9 @@
     background: rgba(255,255,255,0);
     font-size: 16px;
     line-height: 34px;
-    padding:0;
-    margin:0;
-    padding-top:22px;
+    padding: 0;
+    margin-left: 27px;
```
Comment (ericgpks) [inline app/assets/stylesheets/object/user/article/article.scss]: Please fix the order.

File: app/assets/stylesheets/object/user/companies/show.scss [hunk 7]
```diff
@@ -106,48 +114,75 @@
...
+      border: 1px solid #D2D2D2;
+      float: left;
+      height: 68px;
+      margin-left: 30px;
+      margin-right: 20px;
+      width: 68px;
+    } 
+    .p-i-company-detail{
+      &__border-b {
+        padding: 15px 15px;
```
Comment (ericgpks) [inline app/assets/stylesheets/object/user/companies/show.scss]: Please fix the order.

File: app/assets/stylesheets/object/user/about/about.scss [hunk 8]
```diff
@@ -297,13 +269,43 @@
...
 }
 
+
+.p-i-head-line{
+  margin-bottom: 45px;
+  margin-top: 45px;
+  text-align: center;
+  &__line{
+    border-bottom: 5px solid #62BA69;
+    width: 100px;
```
[the remark this answers is not in the document]
Comment (hugh-n) [reply app/assets/stylesheets/object/user/about/about.scss]: This should be ok now.

File: app/assets/stylesheets/object/user/default/default.scss [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+}
+
+@media (max-width: 480px){
+.btn{
+  &__green-big{
+    width: 100%;   
+  }
+  &__orange-big{
+  	font-size: 	12px;
```
[the remark this answers is not in the document]
Comment (hugh-n) [reply app/assets/stylesheets/object/user/default/default.scss]: This should be fixed now.

File: app/views/elements/service/_businessdata.html.haml [hunk 10]
```diff
@@ -1,45 +1,54 @@
...
+              .table-striped.p-i-service__table-btn.row
+                = link_to "詳細を見る", business_path(id: business.id), controller: "businesses", method: :get, class: "btn p-i-service__green2 col-md-5"
+                - if !business.document.url.present? 
+                  = link_to "資料請求リクエスト", request_document_url(id: business.id), controller: "businesses", method: :put, class: "btn p-i-service__orange col-md-5 request_document", remote: true, id: "request_document"
+      - if @count > @PER
+        .p-i-service__pagination
+          = paginate(@businesses)
+  - else
+    .p-i-service__container.container
+      - if params['service_id'].present?
```
[the remark this answers is not in the document]
Comment (hugh-n) [reply app/views/elements/service/_businessdata.html.haml]: I removed it.

File: app/views/admin/companies/_form.html.haml (hunk 1 above)
Comment (hugh-n) [reply app/views/admin/companies/_form.html.haml]: I changed it.

File: app/views/admin/companies/_form.html.haml (hunk 2 above)
Comment (hugh-n) [reply app/views/admin/companies/_form.html.haml]: I changed it.

File: app/views/admin/companies/_member_form.html.haml (hunk 3 above)
Comment (hugh-n) [reply app/views/admin/companies/_member_form.html.haml]: I changed it.

File: app/assets/stylesheets/object/user/article/article.scss (hunk 6 above)
Comment (hugh-n) [reply app/assets/stylesheets/object/user/article/article.scss]: This should be ok now.

File: app/assets/stylesheets/object/user/companies/show.scss (hunk 7 above)
Comment (hugh-n) [reply app/assets/stylesheets/object/user/companies/show.scss]: This should be ok now.

File: app/assets/stylesheets/object/user/service/form.scss (hunk 5 above)
Comment (hugh-n) [reply app/assets/stylesheets/object/user/service/form.scss]: This should be ok now.

**Pull request merged**